XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -10,7 +10,7 @@
     <AssemblyName>AppUIBasics</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>

```

Comment:
Is this required? If so, we should merge into MASTER_19H1 branch instead of MASTER.

Reply:
No, I don't believe that is required.  What is required, however, is that the latest Win 10 SDK be installed to have a fix in the markup compiler.  I'm assuming that means also having the latest OS. 
 I haven't tried using a new SDK on an old OS to see how well it works.  So, it may be that this should still merge into MASTER_19H1?  Without that fix for the markup compiler there would be a runtime error (only in debug) complaining that it was "Unable to cast object of type 'Microsoft.UI.Xaml.Controls.InputKind' to type 'System.Int32'.  See internal issue 20088938.